cmd/influxd/config_test.go

```diff
@@ -274,3 +295,45 @@ lru-cache-size = "200m"
 # they get flushed into backend.
 point-batch-size = 50
 `
+
+func Test_Collectd_ConnectionString(t *testing.T) {
```

Comment:
nit: Can you make the name `TestCollectd_` instead of `Test_Collectd_`? I'm not sure there's a right or wrong way but it's consistent with the rest of the test code when there's no space.